index.html

```diff
@@ -12,6 +12,34 @@
 </head>
 
 <body>
+<!--Header-->
+
+
+
+
+<!--Nav-->
+
+
+
+<!--Section-->
+   <section class="introduction">
+     <p id="intro">
+       This the blue-sky collaborative exercise. It is aimed how to collabrate in git-hub.
+       I find this project very important because it help us in real world project. Codeing 
+       is not how to write your code it include how to communicate with your client, your collaga,
+       and web-designer.
+     </p>
+   </section>
+
+
+<!--Article-->
```

Comment:
I think first aside and then article will come, so arranging elements with float will be easier

Reply:
good point